ui/package.json

```diff
@@ -32,7 +32,7 @@
     "@fortawesome/react-fontawesome": "^0.1.16",
     "@types/color": "^3.0.2",
     "@types/enzyme": "^3.10.10",
-    "@types/jest": "^26.0.21",
```

Comment:
There is a version 28 available but we depend on many packages which use v27, and our code is not yet compatible v28 anyway